docs/reference/modules/asset.md

```diff
@@ -0,0 +1,219 @@
+---
+extends: '@apostrophecms/module'
+---
+
+# `@apostrophecms/asset`
+
+**Alias:** `apos.asset`
+
+<AposRefExtends :module="$frontmatter.extends" />
+
+The `asset` module serves to organize, process, and output all project JavaScript and CSS assets during the build process. In addition, it provides access to modify the project webpack configuration and exposes two CLI tasks for project building and webpack cache clearing. Options are passed through the creation of a `modules/@apostrophecms/asset/index.js` file.
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+| [`refreshOnRestart`](#refreshonrestart) | Boolean | If set to `true`, the browser will refresh on Apostrophe app restart. |
+| [`watch`](#watch) | Boolean | If set to `false`, none of the UI assets will be watched to trigger a restart. |
+| `watchDebounceMs` | Integer | Time in milliseconds to wait before re-triggering a restart on asset change. |
+| [`uploadfs`](#uploadfs) | String \|\| Object | Determines what processor to use for assets during the build process.
```

Comment:
This can't be a string. If specified it must be an object. It's not just about image processing, it's an entire uploadfs options object.

Reply:
Changed.